eng/pipelines/libraries/build-job.yml

```diff
@@ -104,6 +104,13 @@ jobs:
                 sourceFolder: $(Build.SourcesDirectory)/artifacts/bin/testhost
                 targetFolder: $(Build.ArtifactStagingDirectory)/artifacts/bin/testhost
 
+            - ${{ if eq(parameters.runtimeFlavor, 'mono') }}:
+              - task: CopyFiles@2
+                displayName: Prepare lib-runtime-packs folder to publish
+                inputs:
+                  sourceFolder: $(Build.SourcesDirectory)/artifacts/bin/microsoft.netcore.app.runtime.ios-x64
```

Comment:
This will need to be made dynamic so that it works for other platforms as well but I am not sure how it works and how it gets consumed

More information here: https://github.com/dotnet/runtime/commit/2b7852b3d039b7d40f2623a3ec54e7d23eb93790

Reply:
Opened https://github.com/dotnet/runtime/pull/37652 to unblock you.